ndsl/__init__.py

```diff
@@ -1 +1,62 @@
+from .buffer import Buffer
+from .comm.boundary import Boundary, SimpleBoundary
```

Comment:
I don't think these need to be exposed at root

Reply:
Especially not `Buffer` which has a very generic name but should absolutely not be used outside of the internal!